src/main/webapp/search-script.js

```diff
@@ -0,0 +1,84 @@
+function countSearchUnits() {
+  const inputElems = document.getElementsByName("search-units");
+  const arr = []
+  for (let i = 0; i < inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function countAddUnits() {
+  const inputElems = document.getElementsByName("num-units");
+  const arr = []
```

Comment:
arr is never a good variable name by the way. 
Something like addedUnits if that's what you're adding here might be better.